test/commons/math/split-rects.js

```diff
@@ -1,92 +1,98 @@
-describe('splitRects', function () {
-  var splitRects = axe.commons.math.splitRects;
+describe('splitRects', () => {
+  const splitRects = axe.commons.math.splitRects;
   function createRect(x, y, width, height) {
```

Comment:
I don't think we need this if all we're doing is wrap a constructor. I'm not sure but I assume this was before I knew about DOMRect?